src/app/profile/tenant/tenant.component.html

```diff
@@ -42,6 +42,14 @@
           </div>
         </div>
         <div class="form-group">
+          <label for="teamVersion" class="control-label">Team version</label>
```

Comment:
@jstrachan where does this text appear in the UI?

Reply:
on  the advanced OSIO developer/tester page: https://openshift.io/my-user-name/_tenant - its invisible to everyone who doesn't know that magic URL

one day we may even make that magic URL invisible to non 'OSIO testers'

Reply:
ah, thanks

Reply:
background to this 'hidden' page here: https://github.com/fabric8-services/fabric8-tenant#testing-yaml